src/Benchmarks/Data/ApplicationDbContext.cs

```diff
@@ -19,17 +19,27 @@ public class ApplicationDbContext : DbContext
         public ApplicationDbContext(IOptions<AppSettings> appSettings)
         {
             _appSettings = appSettings.Value;
+            UseBatchUpdate = _appSettings.Database != "postgresql";
         }
 
         public DbSet<World> World { get; set; }
 
         public DbSet<Fortune> Fortune { get; set; }
 
+        public bool UseBatchUpdate { get; } 
```

Comment:
NIT: I would make this property return `_appSettings.Database != "postgresql"`, rather than assigning the value in the ctor.  I think it's cleaner since the semantics of the property are in the property getter itself, rather than in the class ctor.